### ISSUE TITLE
Own all land including map edge

### ISSUE THREAD

**Issue by francoism90** [OpenRCT2/OpenRCT2#1520]
I don't know if it is already reported, but for some scenarios it is nice to have no land border.
'In the past' you could perform this by using a trainer.

Is it possible to add this to the game? May even be in the cheat window. :)

Thanks

**Comment by Gymnasiast** [comment]
Currently, you can turn on Sandbox mode in the cheat, which allows you to set land ownership using the Map window. This allows you to own all land, but with far more granular control.
Currently, you can't yet set the edges, though. This limitation is inherited from RCT2, but planned to be taken out.

**Comment by francoism90** [comment]
Thanks for the reply. :)

Would be great if the edges could be removed, without using a trainer. :D
Hope this can integrated in OpenRCT2.

**Comment by Gymnasiast** [comment]
Eventually. Since those are limitations imposed by the file formats of RCT2, we must first decompile the entire game and then replace the formats with our own. Only then can we get rid of those. See here: https://github.com/OpenRCT2/OpenRCT2/wiki/Increasing-of-game-limits

**Comment by IntelOrca** [comment]
The map window needs to be completed first, there are still un-implemented events.

**Comment by francoism90** [comment]
Ah OK, thanks this explained a lot. (Y)

Keep up the good work, and will wait for an update. :)

**Comment by IntelOrca** [comment]
Can be implemented now, no need to be a cheat as its only possible in the scenario editor or sandbox mode.

**Comment by Nubbie** [comment]
Merged Pull-request #2535 now allows you to buy the edges of the map
You can also buy land with sandbox mode enabled

**Comment by francoism90** [comment]
Many thanks! :)
